### PR TITLE
Add base templates

### PR DESCRIPTION
**Pull request by garyservin**
This PR adds base templates to tailor-image, allowing it to build basic images without depending on an external package for the packer templates and ansible playbook.

The ansible playbook can be overridden by specifying the `package` and `provision_file` attributes on the recipe file.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -92,11 +92,11 @@ pipeline {
                             string(credentialsId: 'ansible_vault_password', variable: 'ANSIBLE_VAULT_PASS')]) {
             parent_image = docker.build(parent_image_label,
               "-f tailor-image/environment/Dockerfile --cache-from ${parent_image_label} " +
-              "--build-arg APT_REPO=${params.apt_repo} " +
-              "--build-arg RELEASE_LABEL=${params.release_label} " +
-              "--build-arg RELEASE_TRACK=${params.release_track} " +
-              "--build-arg FLAVOUR=${testing_flavour} " +
-              "--build-arg ORGANIZATION=${organization} " +
+              "--build-arg APT_REPO=s3://locus-tailor-artifacts " +
```
Comment (paulbovbel) [inline Jenkinsfile]: should these be hardcoded?

Comment (garyservin) [reply Jenkinsfile]: :man_facepalming: fixed. Thanks

File: tailor_image/create_image.py [hunk 2]
```diff
@@ -36,22 +36,39 @@ def create_image(name: str, distribution: str, apt_repo: str, release_track: str
...
-    provision_file = recipe[name]['provision_file']
     env = source_file(f'{os.environ["BUNDLE_ROOT"]}/{distro}/setup.bash')
-    template_path = find_package(package, f'image_recipes/{name}/{name}.json', env)
     today = datetime.date.today().strftime('%Y%m%d')
     extra_vars = []  # type: List[Any]
 
+    try:
+        package = recipe[name]['package']
+        provision_file = recipe[name]['provision_file']
+        ansible_path = find_package(package, 'ansible.cfg', env)
```
Comment (paulbovbel) [inline tailor_image/create_image.py]: can you set `env['ANSIBLE_CONFIG'] = ansible_path` immediately after this, instead of the locals hack

**Comment by garyservin** [comment]
> So I think the disconnect here is that I thought you were gonna get rid of the common packer bits from locus_ansible and move them here, so that tailor-image would essentially inject provisioners into a common template for docker and bare metal.

I still want to implement that at some point, this was a first step towards that.

**Pull request merged**